collection_requirements.rst

```diff
@@ -349,6 +349,7 @@ To be included in the `ansible` package, collections must meet the following cri
 
 * `development conventions <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_best_practices.html>`_
 * `Collection requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_ (this document)
+* `Collection Inclusion Criteria Checklist <https://github.com/ansible-collections/overview/blob/main/collection_checklist.md>`_
```

Comment:
Since this is basically a summary of this document, how about indenting it?
```suggestion
  * The `Collection Inclusion Criteria Checklist <https://github.com/ansible-collections/overview/blob/main/collection_checklist.md>`_ covers most of the criteria from this document

```
(most of because it probably misses a few details)

Reply:
Make sense, Thanks @felixfontein!